### ISSUE TITLE
Feature Request: Separate Kana from Kanji in the "Characters Studied" statistic 

### ISSUE THREAD

**Issue by polandreh** [SYtor/Kanji-Dojo#99]
The Characters Studied shows both the number of Kanji practiced as well as the Kana, making the number of kanji studied seem a bit higher than it is.

Could we separate the number into "Kana studied" and "Kanji studied"?

It would help greatly for certifications that require knowing a certain number of kanji.

**Comment by syt0r** [comment]
I can definitely see a value in it for users, maybe there could be a separate counter specifically for kanji. I wonder if it should exclude radicals too, it might be difficult to classify radicals that are not kanji themselves

**Comment by polandreh** [comment]
Yeah, removing radicals would also make sense as they are not kanjis on their own. I've added匕 which shows no definition and is only a radical. I wouldn't consider that one.

斤 and 巾, however, do say they are Jōyō kanji. 禾 says it's a Jinmeiyō kanji. It might not be as simple as just flagging out anything that says "radical".

辶 is not even recognized by the app, so it seems to be an inconsistency with dictionary that maybe the app doesn't need to handle.
